packages/html/test/html-recipes.test.ts

```diff
@@ -126,18 +124,20 @@ describe("recipes with HTML", () => {
 
     await runtime.idle();
 
-    const parent = document.createElement("div");
-    document.body.appendChild(parent);
+    const root = document.getElementById("root")!;
     const cell = result.key(UI);
-    render(parent, cell.get());
+    render(root, cell.get(), renderOptions);
 
+    // Keys are "[object Object]" due to mapping an `Opaque<number>` in handler.
+    // Maybe unintentional(?)
```

Comment:
AFAICT, calling `toString()` on an `Opaque<number>` in a recipe results in a `Prop` value as `[object Object]`, which I believe is "correct" here. If this isn't the case, then I believe the recipe here needs to be corrected